src/languageservice/services/yamlValidation.ts

```diff
@@ -5,7 +5,7 @@
  *--------------------------------------------------------------------------------------------*/
 'use strict';
 
-import { Diagnostic, Position } from 'vscode-languageserver';
+import { Diagnostic, Position } from 'vscode-languageserver-types';
 import { LanguageSettings } from '../yamlLanguageService';
 import { YAMLDocument, YamlVersion } from '../parser/yamlParser07';
 import { SingleYAMLDocument } from '../parser/yamlParser07';
```

Comment:
Not really related to this PR but yamlParser07 is imported multiple times.

Reply:
This could be linted using ESLint rule [import/no-duplicates](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-duplicates.md). Actually this plugin could help maintain more consistency, such as import sorting.